src/box/lua/iproto.lua

```diff
@@ -0,0 +1,30 @@
+-- Sets IPROTO request handler callback (second argument) for the given request
+-- type (first argument, number).
+-- Passing nil as the callback resets the corresponding request handler.
+box.iproto.override = function(request_type, callback)
+    local trigger = require('trigger')
+
+    if request_type == nil then
+        box.error(box.error.PROC_LUA,
+                  'Usage: box.iproto.override(request_type, callback)')
+    end
+    if type(request_type) ~= 'number' then
+        box.error(box.error.PROC_LUA,
+                  ("bad argument #1 to 'override' (number expected, " ..
+                   "got %s)"):format(type(callback)))
+    end
+    if callback ~= nil and type(callback) ~= 'function' then
+        box.error(box.error.PROC_LUA,
+                  ("bad argument #2 to 'override' (function expected, " ..
+                   "got %s)"):format(type(callback)))
+    end
+
+    local event_name = ('box.iproto.override[%d]'):format(request_type)
+    local trigger_name = 'box.iproto.override.lua.user_handler'
```

Comment:
What if a user incidentally has a trigger with the same name?

Reply:
The new trigger will override the old one. That's why I called it "__tarantool__internal__iproto__override__trigger__" in the first PR iteration.

Reply:
In the triggers documentation (which doesn't exist yet) we will write that `tarantool` and `box` root namespaces are reserved for our purposes.